Testscripts/Linux/dpdkTestPmd.sh

```diff
@@ -94,41 +81,37 @@ runTestPmd()
 	# Check testpmd --no-pci if it triggers a kernel crash
 	# SIGKILL(9) is required, as the --no-pci makes testpmd hang
 	# although SIGINT is sent.
-	echo 4096 > /sys/devices/system/node/node0/hugepages/hugepages-2048kB/nr_hugepages \
-		&& echo 1 > /sys/devices/system/node/node0/hugepages/hugepages-1048576kB/nr_hugepages \
-		&&  modprobe -a ib_uverbs mlx4_en mlx4_core mlx4_ib; \
+	# TODO: check nic version and then include drivers: mlx4/mlx5.
+	modprobe -a ib_uverbs mlx4_en mlx4_core mlx4_ib; \
```

Comment:
Need to verify after the module load

Reply:
This might not be a good idea as not all distro have dynamically linked modules some have these modules statically linked. This is just a preventive step to make sure that it is loaded. If not then dpdk would give warning.